src/data/markdown/docs/30 xk6-browser/01 xk6-browser/09 Locator.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Locator"
+excerpt: "xk6-browser: Locator Class"
+---
+
+The Locator API makes it easier to work with dynamically changing elements. Some of the benefits of using it over existing ways to locate an element (e.g. `Page.$()`) include:
+
+- Helps with writing robust tests by finding an element even if the underlying frame navigates.
+- Makes it easier to work with dynamic web pages and SPAs built with Svelte, React, Vue, etc.
+- Enables the use of test abstractions like the Page Object Model (POM) pattern to simplify and organize tests.
+- `strict` mode is enabled for all `locator` methods, which means that if more than one element matches the given selector it will throw an error.
+
+Locator can be created with the [page.locator(selector[, options])](/javascript-api/xk6-browser/page/#page-locator) method.
+
+| Method                                                                                                        | Description                                                                                                                |
+|---------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| <BWIPT id="471"/> [locator.check([options])](/javascript-api/xk6-browser/locator/check)                       | Select the input checkbox.                                                                                                 |
+| <BWIPT id="471"/> [locator.click([options])](/javascript-api/xk6-browser/locator/click)                       | Mouse click on the chosen selector.                                                                                        |
+| <BWIPT id="469"/> [locator.dblclick([options])](/javascript-api/xk6-browser/locator/dblclick)                 | Mouse double click on the chosen selector.                                                                                 |
```

Comment:
Nit: technically, the click is on an element located by the selector.

Let's also update the description on each page.

Same goes for `tap()` below.

Reply:
Resolved in: 22dad9ff4b1a8166a764a18b389fca18538b9449